examples/eSealing/src/lib.rs

```diff
@@ -0,0 +1,444 @@
+//! A contract implementing an eSealing service.
+//!
+//! Only accounts can register a file hash. During the registration, the
+//! timestamp is recorded together with the witness (sender_account) that
+//! registered the file hash.
+//!
+//! Every account can only register each file hash
+//! once. This is because we want to record the time when the witness
+//! (sender_account) first had access to that file. Re-registering a file hash
+//! by the same account would not give us any additional information.
+//!
```

Comment:
Maybe add a bit on the use case.
To my understanding, we want an account to be able to claim to have seen some document (hash) at some point in time, such that others cannot claim the document was created/modified at a later time.

Reply:
We plan to have these two basic functionalities supported for the beginning:
- Upload a file from the computer => register its file hash in the smart contract via the Concordium Browser wallet
Similar to:
http://filesealing.newtrust.co.s3-website-us-east-1.amazonaws.com/
- Upload a file from the computer => retrieve the time-stamp and witness (sender_account) that registered the file hash
Similar to:
https://timestamp.northstake.dk/